ui/components/__tests__/Pagination.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { fireEvent, render, screen, within } from "@testing-library/react";
+import "jest-styled-components";
+import React from "react";
+import { withTheme } from "../../lib/test-utils";
+import Pagination from "../Pagination";
+
+describe("Pagination", () => {
+  const onForward = jest.fn(() => "");
+  const onSkipForward = jest.fn(() => "");
+  const onBack = jest.fn(() => "");
+  const onSkipBack = jest.fn(() => "");
+  const onSelect = jest.fn(() => "");
+  it("has functioning navigation buttons and display text", () => {
+    render(
+      withTheme(
+        <Pagination
+          onForward={onForward}
+          onSkipForward={onSkipForward}
+          onBack={onBack}
+          onSkipBack={onSkipBack}
+          onSelect={onSelect}
+          index={5}
+          length={1}
+          totalObjects={10}
+        />
+      )
+    );
+    const back = screen.getByLabelText("back one page");
+    const skipBack = screen.getByLabelText("skip to first page");
+    const forward = screen.getByLabelText("forward one page");
+    const skipForward = screen.getByLabelText("skip to last page");
+    const displayText = screen.getByText(/6 - 6 out of 10/);
+    fireEvent.click(forward);
+    expect(onForward).toHaveBeenCalledTimes(1);
+    fireEvent.click(skipForward);
+    expect(onSkipForward).toHaveBeenCalledTimes(1);
+    fireEvent.click(skipBack);
+    expect(onSkipBack).toHaveBeenCalledTimes(1);
+    fireEvent.click(back);
+    expect(onBack).toHaveBeenCalledTimes(1);
+    expect(displayText).toBeTruthy();
```

Comment:
What should this display text actually be? `.toBeTruthy()` will return ok on literally any string.

Reply:
Yes but the test will fail if it does not match /6 - 6 out of 10/ because it won't find the string at all